synapse/storage/databases/main/stream.py

```diff
@@ -985,7 +996,9 @@ async def get_all_new_events_stream(
             the `current_id`).
         """
 
-        def get_all_new_events_stream_txn(txn):
+        def get_all_new_events_stream_txn(
+            txn: LoggingTransaction,
+        ) -> Tuple[int, Collection[str]]:
```

Comment:
Similarly to earlier: why `Collection` here rather than `List`?

Reply:
No reason. I have changed it to `List`.